checkstyle.properties

```diff
@@ -1,16 +1,24 @@
 category=External Analysers
 description=Provide Checkstyle rules for Java projects
 homepageUrl=https://github.com/checkstyle/sonar-checkstyle
-archivedVersions=10.5,10.6.0,10.7.0,10.8.1,10.9.3,10.11.0,10.12.0,10.12.1,10.12.3,10.12.5,10.14.1,10.14.2,10.15.0,10.16.0,10.17.0,10.19.0
-publicVersions=10.20.1
+archivedVersions=10.5,10.6.0,10.7.0,10.8.1,10.9.3,10.11.0,10.12.0,10.12.1,10.12.3,10.12.5,10.14.1,10.14.2,10.15.0,10.16.0,10.17.0,10.19.0,10.20.1
+publicVersions=10.20.2
 
 defaults.mavenGroupId=org.sonarsource.checkstyle
 defaults.mavenArtifactId=sonar-checkstyle-plugin
 
+10.20.2.description=Upgrade to Checkstyle 10.20.2
+10.20.2.sqs=[10.8,LATEST]
+10.20.2.sqcb=[24.12,LATEST]
+10.20.2.sqVersions=[9.9,10.7.*]
+10.20.2.date=2024-12-07
+10.20.2.changelogUrl=https://github.com/checkstyle/sonar-checkstyle/releases/tag/10.20.2
+10.20.2.downloadUrl=https://github.com/checkstyle/sonar-checkstyle/releases/download/10.20.2/checkstyle-sonar-plugin-10.20.2.jar
+
 10.20.1.description=Upgrade to Checkstyle 10.20.1
-10.20.1.sqs=[10.8,LATEST]
-10.20.1.sqcb=[24.12,LATEST]
-10.20.1.sqVersions=[9.9,10.7]
+10.20.1.sqs=[10.8,10.8.*]
+10.20.1.sqcb=[24.12,10.8.*]
```

Comment:
I didn't pick up on this before merging & I'll correct it in `main`. The ranges don't cross. 10.8 doesn't come after (or before) 24.12. I'll update this range to end with 24.12.*